src/main/java/use_cases/hazards/MazeEnemies.java

```diff
@@ -1,5 +1,6 @@
 package use_cases.hazards;
 
+import adapters.hazards.IEnemyRequestModel;
```

Comment:
As I mentioned earlier, use cases layer should also be independent of the adapters layer.